packages/ariakit/src/focus-trap/__examples__/focus-trap/test.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  click,
+  focus,
+  getByLabelText,
+  getByRole,
+  getByText,
+  press,
+  render,
+} from "ariakit-test-utils";
+import Example from ".";
+
+test("correctly traps focus", async () => {
+  // Focus trap is on by default in the Example
+  render(
+    <div>
+      <Example />
+      <div tabIndex={0}>Outside</div>
+    </div>
+  );
+  focus(getByRole("checkbox"));
```

Comment:
I think we should use something that better describes a user action here, like `await press.Tab()` or `await click()`.